en-US/firefox/new/horizon.lang

```diff
@@ -123,9 +130,9 @@ Firefox for other platforms
 Download Firefox <span>for Windows</span>
 
 
-## TAG: firefox_new_other_platforms
-;Download Firefox <span>for OS X</span>
-Download Firefox <span>for OS X</span>
+## TAG: macos_branding
```

Comment:
My thinking on this is that since this has not been exposed, so I thought we can remove it all together.